packages/fluentui/react-northstar-prototypes/src/prototypes/VirtualizedStickyTree/VirtualStickyTree.tsx

```diff
@@ -31,6 +32,12 @@ export interface VirtualStickyTreeProps
   itemSize: number;
   /** height of 1st level sticky tree item */
   stickyItemSize: number;
+
+  /**
+   * A function that converts an item to string.
+   * Used for keyboard navigation based on the first letter of an item's text content
+   */
+  itemToString?: (item: ShorthandValue<TreeItemProps>) => string;
```

Comment:
should we adopt a similar API in `Tree` for a bit of consistency ?

Reply:
I don't think so. This is for `VirtualTree` because items might not be mounted. For classic trees, we can just get text content from DOM. Plus `VirtualTree` is just a prototype, and `Tree` is a component, so there's more at stake when introducing new props for `Tree`. I think we could keep it this way, unless someone specifically ask for such function in `Tree`.

Reply:
Alright, I would assume that feature engineers can have some whacky ideas about text search, but you're right we shouldn't open that can of worms yet